### DISCUSSION TITLE
Cannot jump to bookmark with `A after restart.

### DISCUSSION THREAD

**Discussion by nicu-chiciuc** [JetBrains/ideavim#409] [General]
Let's say I create a bookmark with mA, PyCharm will show A near the line. I can jump to it from whatever file using `A. If I restart PyCharm the bookmark is still visible in the editor but I cannot jump to it using `A.

**Comment by AlexPl292** [comment]
Hi, please follow this issue: https://youtrack.jetbrains.com/issue/VIM-2026
